server/server.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const path = require('path');
+const app = express();
+const port = process.env.PORT || 3001;
+
+app.listen(port, () => {
+  console.log(`Example app listening at http://localhost:${port}`);
+});
+
+app.use(express.static(path.join(__dirname, '..', 'build')));
+
+app.get('/', (req, res) => {
+  res.sendFile(path.join(__dirname, '..', 'build', 'index.html'));
+});
```

Comment:
would you mind telling me what the server is for? 😮

Reply:
It's what Heroku uses to run the App. 
https://shielded-savannah-94425.herokuapp.com/